LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2Effect_Sustain_LW.uc

```diff
@@ -0,0 +1,48 @@
+class X2Effect_Sustain_LW extends X2Effect_Sustain;
+
+function bool PreDeathCheck(XComGameState NewGameState, XComGameState_Unit UnitState, XComGameState_Effect EffectState)
+{
+	local X2EventManager EventMan;
+	local UnitValue SustainValue;
+	local int Index, PercentChance, RandRoll;
+    local UnitValue OverKillDamage;
+	if( !UnitState.IsAbleToAct(true) )
+	{
+		// Stunned units may not go into Sustain
+		return false;
+	}
+
+	if (UnitState.GetUnitValue(default.SustainUsed, SustainValue))
+	{
+		if (SustainValue.fValue > 0)
+			return false;
+    }
```

Comment:
You managed to introduce some spaces for indent somehow

Reply:
Wait, What? Where? Don't understand that one

Reply:
On several lines, hence why the curly braces are misaligned above in GitHub. One of the local variable declarations also has spaces, as does the `UnitState.GetUnitValue()` line below here. Pretty obvious if you look at the file on GitHub.